### PR TITLE
Disable medicaid tax credit links

### PR DESCRIPTION
**Pull request by jacobkagon**
### Master Pivotal ticket
* https://www.pivotaltracker.com/n/projects/2481183/stories/178953781 

### Child Redmine ticket(s)
* ()
* ()

### Local build result

```
bundle exec rake parallel:spec[4]
bundle exec cucumber
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: app/views/insured/families/inbox.html.erb [hunk 1]
```diff
@@ -16,8 +16,9 @@
 <br/>
 <% else %>
 &nbsp;
-
+<% unless !medicaid_tax_credits_link_is_enabled? %>
 <div>
   <%= render partial: "navigate_to_curam_for_documents", :locals => {display_text: "If you applied for Medicaid and tax credit savings, view additional messages"} %>
+<% end %>
```
Comment (bill-transue) [inline app/views/insured/families/inbox.html.erb:22]: This needs to go over line 23 so the `</div>` is conditionally added to the view with the opening `<div>`.

File: app/views/insured/families/verification.html.erb [hunk 2]
```diff
@@ -12,7 +12,8 @@
   <% end %>
 </div>
 &nbsp;
-
+ <% if medicaid_tax_credits_link_is_enabled? %>
 <div>
   <%= render partial: "navigate_to_curam_for_documents", :locals => {display_text: "If you applied for Medicaid and tax credit savings, view additional documents"} %>
 </div>
```
Comment (bill-transue) [inline app/views/insured/families/verification.html.erb:18]: These lines need to be indented within the `if` block.

File: app/views/insured/families/inbox.html.erb [hunk 3]
```diff
@@ -16,8 +16,9 @@
 <br/>
 <% else %>
 &nbsp;
-
+<% unless !medicaid_tax_credits_link_is_enabled? %>
 <div>
   <%= render partial: "navigate_to_curam_for_documents", :locals => {display_text: "If you applied for Medicaid and tax credit savings, view additional messages"} %>
```
Comment (bill-transue) [inline app/views/insured/families/inbox.html.erb:21]: Indent these lines and the closing `</div>` below within the `unless` block.

**Review by bill-transue — CHANGES_REQUESTED** [review]
Minor changes needed to the view but good work overall @jacobkagon.

**Review by bill-transue — APPROVED** [review]
Good work @jacobkagon, this will prevent the medicaid tax credit links from showing up when the feature is turned off.

**Review by bill-transue — APPROVED** [review]
Good work @jacobkagon, this should make the Medicaid tax credit links a configurable item and sets the ME and DC stored configs and the DC committed configs.

**Comment by jacobkagon** [comment]
The contents of this branch have already been merged into trunk in this PR: https://github.com/ideacrew/enroll/pull/620/

**Pull request closed without merging**